README.md.tt

```diff
@@ -97,13 +97,6 @@ docker-compose -f docker-compose.test.yml run test bin/bundle exec spec [rspec-p
 ```
 
 ### Automated Code Review Setup
+- Add a bot (i.e. `team-nimblehq`) to this repository or to the organization. The bot requires permission level “Write” to be able to set a PR’s status.
 
-* Make sure that the config file `pronto.yml` for pronto has been added as a configuration file.
-
-* In the CI build script, add the following command:
-
-```sh
-if ([ $BRANCH_NAME != 'master' ] && [ $BRANCH_NAME != 'development' ]); then \
-  (echo "Running pronto"; bundle exec pronto run -f bitbucket_pr -c origin/development);  else \ (echo "Escaping pronto"); \
-fi
-```
+- Create a token from bot account with public_repo scope, and use it to create a DANGER_GITHUB_API_TOKEN secret on this repository
```

Comment:
Do we need to add the `DANGER_GITHUB_API_TOKEN` to CI?

Reply:
Yes. The access token of the `BOT` account needs to be added in CI

Reply:
It's a bit confuse with another comment https://github.com/nimblehq/rails-templates/pull/195/files#r472007900 ☁️ Could you please double check it again @noor-ahmed 🤔

Reply:
We updated with a clearer Readme.